Objects/codeobject.c

```diff
@@ -1110,33 +1350,58 @@ static PyMemberDef code_memberlist[] = {
     {"co_argcount",     T_INT,          OFF(co_argcount),        READONLY},
     {"co_posonlyargcount",      T_INT,  OFF(co_posonlyargcount), READONLY},
     {"co_kwonlyargcount",       T_INT,  OFF(co_kwonlyargcount),  READONLY},
-    {"co_nlocals",      T_INT,          OFF(co_nlocals),         READONLY},
     {"co_stacksize",T_INT,              OFF(co_stacksize),       READONLY},
     {"co_flags",        T_INT,          OFF(co_flags),           READONLY},
     {"co_code",         T_OBJECT,       OFF(co_code),            READONLY},
     {"co_consts",       T_OBJECT,       OFF(co_consts),          READONLY},
     {"co_names",        T_OBJECT,       OFF(co_names),           READONLY},
-    {"co_varnames",     T_OBJECT,       OFF(co_varnames),        READONLY},
-    {"co_freevars",     T_OBJECT,       OFF(co_freevars),        READONLY},
-    {"co_cellvars",     T_OBJECT,       OFF(co_cellvars),        READONLY},
     {"co_filename",     T_OBJECT,       OFF(co_filename),        READONLY},
     {"co_name",         T_OBJECT,       OFF(co_name),            READONLY},
     {"co_firstlineno",  T_INT,          OFF(co_firstlineno),     READONLY},
     {"co_linetable",    T_OBJECT,       OFF(co_linetable),       READONLY},
     {"co_exceptiontable",    T_OBJECT,  OFF(co_exceptiontable),  READONLY},
+    {"co_fastlocalnames",    T_OBJECT,  OFF(co_fastlocalnames),  READONLY},
```

Comment:
Don't make this public, at least not for now.

Reply:
We need it for the dis module.

Reply:
Why can't dis use the old API?

Reply:
I've made it a private getter function.

Reply:
> Why can't dis use the old API?

The main reason is that, once #26396 lands, the opargs will no longer match co_varnames + co_cellvars + co_freevars.  It made sense to switch dis to use co_fastlocalnames in this PR rather than that one.

(Also, by using co_fastlocalnames we avoid generating co_varnames, etc., if they weren't already.)

Reply:
I not worried about the performance of `dis`, so generating co_varnames, etc., is fine.
My main concern is that code using the old API should still work, if at all possible.

Reply:
Yes, that's fine.